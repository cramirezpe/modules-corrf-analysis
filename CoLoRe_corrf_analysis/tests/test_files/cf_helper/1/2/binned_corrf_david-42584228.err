.....70%..................50%.40%....................20%......10%............ Time taken =   0.045 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 4,4,3. countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (4, 4, 3); avg_np = 9.14e+03. Boosting bin refine factor - should lead to better performance
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 6,6,3.  Time taken =   0.038 sec
Using AVX kernel
0%30%......60%........80%.........90%.....100% done. Time taken = 18.893 secs
100% done. Time taken = 38.072 secs
........90%........80%..................50%..40%.......30%..........10%... Time taken =   0.032 sec
 Time taken =   0.023 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 6,6,3. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 4,4,3.  Time taken =   0.020 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (4, 4, 3); avg_np = 9.14e+03. Boosting bin refine factor - should lead to better performance
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 6,6,3. 0%...........20%................60%70%...........90%...50%.............30%.........40%......... Time taken =   0.028 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 6,6,3. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 4,4,3.  Time taken =   0.021 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (4, 4, 3); avg_np = 9.39e+03. Boosting bin refine factor - should lead to better performance
0%......10%.........20%........60%.........70%.........80%.........100% done. Time taken = 19.699 secs
