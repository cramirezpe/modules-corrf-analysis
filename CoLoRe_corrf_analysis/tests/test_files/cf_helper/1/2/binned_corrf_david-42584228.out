verson 0.12
DEBUG:__main__:main:data:	/global/cscratch1/sd/cramirez/NBodyKit/hanyu_david_box/catalogues//colore_2_full_rsd.fits
randoms:	/global/cscratch1/sd/cramirez/NBodyKit/david_box/randoms_rsd.fits
out_dir:	/global/cscratch1/sd/cramirez/NBodyKit/hanyu_david_box/binned_analysis_with_error//nside_2/rsd/0.1_200/0.1_0.3//2
nthreads:	64
mu_max:	1.0
nmu_bins:	100
min_bin:	0.1
max_bin:	200.0
n_bins:	41
norsd:	False
zmin:	0.1
zmax:	0.3
drq_format:	True
zmin_covd:	0.01
zmax_covd:	1.5
zstep_covd:	0.005
random_downsampling:	1
pixel_mask:	[2]
nside:	2
log_level:	DEBUG
DEBUG:__main__:main:Data files:/global/cscratch1/sd/cramirez/NBodyKit/hanyu_david_box/catalogues//colore_2_full_rsd.fits
DEBUG:__main__:main:Random files:/global/cscratch1/sd/cramirez/NBodyKit/david_box/randoms_rsd.fits
INFO:__main__:main:Reading file:
	/global/cscratch1/sd/cramirez/NBodyKit/hanyu_david_box/catalogues//colore_2_full_rsd.fits
		/global/cscratch1/sd/cramirez/NBodyKit/david_box/randoms_rsd.fits
DEBUG:__main__:main:Length of data cat: 428941714
DEBUG:__main__:main:Length of rand cat: 428941714
DEBUG:__main__:main:Applying pixel mask
DEBUG:__main__:main:Length of data after pixel mask: 8966997
DEBUG:__main__:main:Length of rand after pixel mask: 8932877
DEBUG:__main__:main:Length of data after z mask: 438550
DEBUG:__main__:main:Length of rand after z mask: 450612
INFO:__main__:main:Starting computation for current file:
DEBUG:__main__:main:Relative ellapsed time: 77.96087598800659
